collection-scripts/gather_admission_webhooks

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="/must-gather"
+WEB_HOOK_PATH="${BASE_COLLECTION_PATH}/web_hooks/"
+
+mkdir ${WEB_HOOK_PATH}
```

Comment:
maybe `mkdir -p` ? Or are we sure that dir does not exist?

Reply:
fixed.